events/templates/meetings/create_meeting.html

```diff
@@ -0,0 +1,110 @@
+            {{optional_meetings_formset.management_form}}
+            <div id="optional-meeting-dates-form-list">
+                {% if formset_meeting_data %}
+                    {% for form in formset_meeting_data %}
+                        <div id="form-{{form.id}}" class="optional-meeting-dates-form">
+                            <p>
+                                <label for="id_optional_meetings-{{form.id}}-date_time_start">Date time start:</label>
+                                <input type="datetime-local" value="{{form.date_time_start}}" name="optional_meetings-{{form.id}}-date_time_start" id="id_optional_meetings-{{form.id}}-date_time_start">
+                            </p>
+                            <p>
+                                <label for="id_optional_meetings-{{form.id}}-date_time_end">Date time end:</label>
+                                <input type="datetime-local" value="{{form.date_time_end}}" name="optional_meetings-{{form.id}}-date_time_end" id="id_optional_meetings-{{form.id}}-date_time_end">
+                            </p>
+                        </div>
+                    {% endfor %}
+                {% endif %}
+            </div>
+            
+            <div id="empty-optional-meeting-dates-form" class="hidden">{{ optional_meetings_formset.empty_form.as_p }}</div>
+            <button id="add-more-optional-meeting-dates" type="button" class="btn btn-dark">Add more dates</button>
+            
+            <br/>
+            <br/>
+            {% if meeting_participants_formset.non_form_errors %}
+                {{ meeting_participants_formset.non_form_errors }}
+            {% endif %}
+
+            {% if meeting_participants_formset.errors %}
+                {% for error in meeting_participants_formset.errors %}
+                    {{ error|escape }}
+                {% endfor %}
+            {% endif %}
+            <br/>
+            <h4>Participants:</h4>
+            {{meeting_participants_formset.management_form}}
+            <div id="meeting-participants-form-list">
+                {% if formset_participant_data %}
+                    {% for form in formset_participant_data %}
+                        <div id="form-{{form.id}}" class="meeting-participants-form">
+                            <p>
+                                <label for="id_participants-{{form.id}}-participant_email">Participant email:</label>
+                                <input type="email" value="{{form.email}}" name="participants-{{form.id}}-participant_email" id="id_participants-{{form.id}}-participant_email">
+                            </p>
+                        </div>
+                    {% endfor %}
+                {% endif %}
+            </div>
+
+            <div id="empty-meeting-participants-form" class="hidden">{{ meeting_participants_formset.empty_form.as_p }}</div>
+            <button id="add-more-meeting-paricipants" type="button" class="btn btn-dark">Add more participants</button>
```

Comment:
```suggestion
            <button id="add-more-meeting-paricipants" type="button" class="btn btn-dark">
                <i class="fas fa-user-plus" aria-hidden="true" style='margin-right: 10px; font-size:15px; color: black;'></i>
                Add more participants
            </button>
```

Maybe you can add an icon that represents what the button does 😄

Reply:
Thanks, done 😄